### DISCUSSION TITLE
Gruppenzusammensetzung Data Busters

### DISCUSSION THREAD

**Discussion by Foxglovve** [irgroup-classrooms/dis08-2021#51] [Q&A] [answered]
Hallo Herr Schaer,

es gibt leider etwas Chaos in unserem Gruppenraum. Einige Personen sind doppelt drin, eine gehört nicht hinein und zwei fehlen noch und können nicht rein, da der Raum voll ist. Könnten Sie das bitte richten?

Unsere Members sind:
Foxglovve
floriflowers
jmmichev
MehmetCagferoglu

während huifenglee nicht zu unserer Gruppe gehört.

Sorry und vielen Dank!

**Comment by phschaer** [comment, marked as answer]
fixed
